Fedora-35/Dockerfile

```diff
@@ -78,7 +78,7 @@ RUN npm install -g npm \
 #Building qemu from source:
 FROM build AS test
 ARG QEMU_URL="https://gitlab.com/qemu-project/qemu.git"
-ARG QEMU_BRANCH="v7.2.0"
```

Comment:
Can you update the version in the readme? https://github.com/tianocore/containers/blob/main/Fedora-37/Readme.md

Reply:
Done.